<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToNgueLP_MW</class>
 <widget class="QMainWindow" name="ToNgueLP_MW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToNgueLP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="corpusTabs">
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Corpus X</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>216</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>216</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="searchCase">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Search case?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="casesList">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="casesCount">
                 <property name="text">
                  <string>Total: 10</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTabWidget" name="casesWorkingArea">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Case 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTextEdit" name="sourceDoc">
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;By Earth911&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Wednesday, June 20, 2007&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#00ff00;&quot;&gt;In United States, about half the air pollution comes from cars and trucks. Two important ways to reduce air pollution are to drive less, even a little less, and to drive smart. Taking fewer trips in your car or truck helps cut air pollution. And adopting smart driving habits reduces your car’s emissions.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#00ff00;&quot;&gt;Driving less doesn’t mean you have to stay home. Try combining driving with alternative modes of transportation&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Carpool.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Walk or ride a bicycle.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Shop by phone or mail.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ride public transit.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Driving smart keeps pollution at a minimum&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Accelerate gradually.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use cruise control on the highway.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Obey the speed limit.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Combine your errands into one trip.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Keep your car tuned and support the smog check program.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Don’t top off at the gas pumps.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Replace your car’s air filter.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Keep your tires properly inflated.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;What about smoking vehicles?&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Contact the EPA’s Office of Air Quality Planning &amp;amp; Standards,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When&lt;span style=&quot; color:#000000;&quot;&gt; shopping for your next car&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Look for the most efficient, lowest polluting model–or even use either a non-polluting car or zero emission vehicle. Check out these Web sites for information that will help you identify clean and fuel efficient vehicles in any part of the country:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="sourceInfo">
                <property name="text">
                 <string>id = 278, src_name = src-doc45, length = 570 chars 70 words 5 sentences, orig_offset = 329</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="suspDoc">
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;At this stage of research, scientists can not firmly link global warming to changes in circulation patterns. But one study by British and Italian scientists, reported recently in the journal Nature, examined data on atmospheric circulation over the  last 50 years  in the  Northern Hemisphere . &lt;span style=&quot; font-style:italic; color:#ff0000;&quot;&gt;The researchers looked to see whether there had been any change in frequency of large-scale circulation regimes. The data revealed that there had : in recent decades, patterns had shifted to favor a warmer climate for most land areas in the  Northern Hemisphere . In a general warming of the globe, the air above those land areas would be expected to warm more than that above the oceans, because of differing abilities to absorb heat.&lt;/span&gt; But the authors of the new study postulate that the change in circulation patterns is the main reason the land has lately warmed so much. That is, they say, the climate of the hemisphere shifts naturally back and forth between two dominant regimes. Both the  North Atlantic  and  North Pacific  oscillations are believed to be affected by this shift.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="suspInfo">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;id = 367, src_name = susp-doc69, length = 489 chars 67 words 5 sentences, orig_offset = 125&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
           <zorder></zorder>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Case 2</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Compare Cases</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Corpus Y</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCorpus">
    <property name="title">
     <string>Corpus</string>
    </property>
    <addaction name="actionLoad_New_Corpus"/>
    <addaction name="actionAdd_New_Corpus_Doc"/>
    <addaction name="actionSave_Corpus"/>
    <addaction name="actionCorpus_Information"/>
    <addaction name="actionSelect_Parser"/>
    <addaction name="actionSuggest_Parser"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMain_View"/>
    <addaction name="actionComparison_View"/>
    <addaction name="actionDictionary_View"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="title">
     <string>Scripts</string>
    </property>
    <addaction name="actionConvert_Documents"/>
    <addaction name="actionVerify_Corpus_and_Parser"/>
    <addaction name="actionGenerate_XML_Output"/>
    <addaction name="actionCompare_XMLs"/>
    <addaction name="actionUpdate_XMLs_Info"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_ToNgueLP"/>
    <addaction name="actionUser_s_Manual"/>
   </widget>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
   </widget>
   <addaction name="menuCorpus"/>
   <addaction name="menuTools"/>
   <addaction name="menuConfig"/>
   <addaction name="menuScripts"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_New_Corpus"/>
   <addaction name="actionAdd_New_Corpus_Doc"/>
   <addaction name="actionCorpus_Information"/>
   <addaction name="actionClose_Corpus"/>
   <addaction name="actionAdd_Algorithm_Output"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_New_Case"/>
   <addaction name="actionEdit_Case"/>
   <addaction name="actionValidate_Case"/>
   <addaction name="action_Re_Define_Boundaries"/>
   <addaction name="actionView_Annotation"/>
  </widget>
  <action name="actionLoad_New_Corpus">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/menu_corpus.png</normaloff>:/toolbar/icons/menu_corpus.png</iconset>
   </property>
   <property name="text">
    <string>Load New Corpus</string>
   </property>
  </action>
  <action name="actionAdd_New_Corpus_Doc">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/add_corpus_doc.png</normaloff>:/toolbar/icons/add_corpus_doc.png</iconset>
   </property>
   <property name="text">
    <string>Add New Corpus Doc</string>
   </property>
  </action>
  <action name="actionCorpus_Information">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/corpus_info.png</normaloff>:/toolbar/icons/corpus_info.png</iconset>
   </property>
   <property name="text">
    <string>Corpus Information</string>
   </property>
  </action>
  <action name="actionClose_Corpus">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/close_corpus.png</normaloff>:/toolbar/icons/close_corpus.png</iconset>
   </property>
   <property name="text">
    <string>Close Corpus</string>
   </property>
  </action>
  <action name="actionAdd_Algorithm_Output">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/add_algorithm_output.png</normaloff>:/toolbar/icons/add_algorithm_output.png</iconset>
   </property>
   <property name="text">
    <string>Add Algorithm Output</string>
   </property>
  </action>
  <action name="actionAdd_New_Case">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/add_case.png</normaloff>:/toolbar/icons/add_case.png</iconset>
   </property>
   <property name="text">
    <string>Add New Case</string>
   </property>
  </action>
  <action name="actionEdit_Case">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/edit_case.png</normaloff>:/toolbar/icons/edit_case.png</iconset>
   </property>
   <property name="text">
    <string>Edit Case</string>
   </property>
  </action>
  <action name="actionValidate_Case">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/validate_case.png</normaloff>:/toolbar/icons/validate_case.png</iconset>
   </property>
   <property name="text">
    <string>Validate Case</string>
   </property>
  </action>
  <action name="action_Re_Define_Boundaries">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/redefine_boundaries.png</normaloff>:/toolbar/icons/redefine_boundaries.png</iconset>
   </property>
   <property name="text">
    <string>(Re)Define Boundaries</string>
   </property>
  </action>
  <action name="actionView_Annotation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/view_annotation.png</normaloff>:/toolbar/icons/view_annotation.png</iconset>
   </property>
   <property name="text">
    <string>View Annotation</string>
   </property>
  </action>
  <action name="actionSave_Corpus">
   <property name="text">
    <string>Save Corpus</string>
   </property>
  </action>
  <action name="actionSelect_Parser">
   <property name="text">
    <string>Select Parser</string>
   </property>
  </action>
  <action name="actionSuggest_Parser">
   <property name="text">
    <string>Suggest Parser</string>
   </property>
  </action>
  <action name="actionMain_View">
   <property name="text">
    <string>Main View</string>
   </property>
  </action>
  <action name="actionComparison_View">
   <property name="text">
    <string>Comparison View</string>
   </property>
  </action>
  <action name="actionDictionary_View">
   <property name="text">
    <string>Dictionary View</string>
   </property>
  </action>
  <action name="actionConvert_Documents">
   <property name="text">
    <string>Convert Documents</string>
   </property>
  </action>
  <action name="actionVerify_Corpus_and_Parser">
   <property name="text">
    <string>Verify Corpus and Parser</string>
   </property>
  </action>
  <action name="actionGenerate_Output">
   <property name="text">
    <string>Generate Output</string>
   </property>
  </action>
  <action name="actionCompare_Parser_Output_and_XML_Output">
   <property name="text">
    <string>Compare Parser Output and XML Output</string>
   </property>
  </action>
  <action name="actionGenerate_XML_Output">
   <property name="text">
    <string>Generate XML Output</string>
   </property>
  </action>
  <action name="actionCompare_Parser_and_XML_Output">
   <property name="text">
    <string>Compare Parser and XML Output</string>
   </property>
  </action>
  <action name="actionCompare_XMLs">
   <property name="text">
    <string>Compare XMLs</string>
   </property>
  </action>
  <action name="actionUpdate_XMLs_Info">
   <property name="text">
    <string>Update XMLs Info</string>
   </property>
  </action>
  <action name="actionAbout_ToNgueLP">
   <property name="text">
    <string>About ToNgueLP</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionUser_s_Manual">
   <property name="text">
    <string>User's Manual</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
